src/core/model/Element.h

```diff
@@ -47,6 +47,7 @@ class Element: public Serializable {
     virtual void move(double dx, double dy);
     virtual void scale(double x0, double y0, double fx, double fy, double rotation, bool restoreLineWidth) = 0;
     virtual void rotate(double x0, double y0, double th) = 0;
+    virtual void axis_reflect(double x0, double y0, bool axis) = 0;
```

Comment:
I do not understand the "no-op" comment. My understanding was that the declaration was `virtual` because `Element` 
class is inherited by other classes. Could you please give some more details about how this "no-op" implementations should be accomplished? Thanks.

Reply:
Sorry, I wasn't very clear. We want to leave it as virtual, because we want specific element types (e.g., stroke) to be able to override. But most of the concrete implementations of `axis_reflect` are basically `void axis_reflect(double x0, double y0, bool axis) {}` ("no-op" implementation--it doesn't do anything). Thus, it would be better to implement `axis_reflect` in `Element` as the "no-op". Then subclasses can override `axis_reflect` if they need to actually perform the reflection.

```c++
// Default implementation of reflection is no-op for all elements.
virtual void Element::axis_reflect(double x0, double y0, bool axis) { /* no-op */ }

// Stroke needs to perform reflection, so override.
void Stroke::axis_reflect(double x0, double y0, bool axis) override;

// In the future, we could allow images to be reflected, in which case we can add an override.
void Image::axis_reflect(double x0, double y0, bool axis) override;

// other elements don't need to override axis_reflect because they don't actually perform reflection
```

Reply:
Done in dffd514